src/components/search/SearchResults.tsx

```diff
@@ -335,8 +339,8 @@ const SearchResults: FunctionComponent<SearchResultsProps> = ({ serverId = windo
                         cardOptions={{ coverImage: true }}
                     />
 
-                    {allEmpty && query && !isLoading && (
-                        <div className='sorry-text'>{globalize.translate('SearchResultsEmpty', query)}</div>
+                    {allEmpty && debouncedQuery && !isLoading && (
+                        <div className='sorry-text'>{globalize.translate('SearchResultsEmpty', debouncedQuery)}</div>
```

Comment:
displaying no result it bit off
```suggestion
                        <div className='noItemsMessage centerMessage'>{globalize.translate('SearchResultsEmpty', debouncedQuery)}</div>
```